WordPress/Classes/ViewRelated/Post/PostSharingController.swift

```diff
@@ -145,3 +147,51 @@ import SVProgressHUD
 
     typealias PopoverAnchor = UIPopoverPresentationController.PopoverAnchor
 }
+
+private class CopyLinkActivity: UIActivity {
+    var activityItems = [Any]()
+    private var url = URL(string: "")
+
+    override var activityTitle: String? {
+        return NSLocalizedString(
+            "share.sheet.copy.link.title",
+            value: "Copy Link",
+            comment: "Title for the \"Copy Link\" action in Share Sheet."
+        )
+    }
+
+    override var activityImage: UIImage? {
+        return UIImage(systemName: "doc.on.doc")
+    }
+
+    override var activityType: UIActivity.ActivityType? {
+        return UIActivity.ActivityType(rawValue: "copy.link.activity")
+    }
+
+    override class var activityCategory: UIActivity.Category {
+       return .action
+   }
+
+    override func canPerform(withActivityItems activityItems: [Any]) -> Bool {
+        for activityItem in activityItems {
+           if let _ = activityItem as? URL {
+              return true
+           }
+        }
+        return false
+    }
+
+   override func prepare(withActivityItems activityItems: [Any]) {
+       for activityItem in activityItems {
+           if let url = activityItem as? URL {
+               self.url = url
+           }
+       }
+       self.activityItems = activityItems
+   }
+
+   override func perform() {
+    UIPasteboard.general.string = url?.absoluteString
+    activityDidFinish(true)
```

Comment:
```suggestion
       UIPasteboard.general.string = url?.absoluteString
       activityDidFinish(true)
```

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/23007/commits/cc64dafcf82573f494932440b664469a721d618d